docs/content/en/sentry/options.md

```diff
@@ -0,0 +1,251 @@
+Normally, setting required DSN information would be enough.
+
+### dsn
+- Type: `String`
+  - Default: `process.env.SENTRY_DSN || ''`
+  - If no `dsn` is provided, Sentry will be initialised, but errors will not be logged. See [#47](https://github.com/nuxt-community/sentry-module/issues/47) for more information about this.
+
+### lazy
+- Type: `Boolean` or `Object`
+  - Default: `false`
+  - Load Sentry lazily so it's not included in your main bundle
+  - If `true` then the default options will be used:
+  ```js
+    {
+      injectMock: true,
+      injectLoadHook: false,
+      mockApiMethods: true,
+      chunkName: 'sentry',
+      webpackPrefetch: false,
+      webpackPreload: false
+    }
+  ```
+    - **injectMock**
+      - Type: `Boolean`
+        - Default: `true`
+        - Whether a Sentry mock needs to be injected that captures any calls to `$sentry` API methods while Sentry has not yet loaded. Captured API method calls are executed once Sentry is loaded
+      > When `injectMock: true` this module will also add a window.onerror listener. If errors are captured before Sentry has loaded then these will be reported once Sentry has loaded using sentry.captureException
+        ```js
+        // pages/index.vue
+        beforeMount() {
+          // onNuxtReady is called _after_ the Nuxt.js app is fully mounted,
+          // so Sentry is not yet loaded when beforeMount is called
+          // But when you set injectMock: true this call will be captured
+          // and executed after Sentry has loaded
+          this.$sentry.captureMessage('Hello!')
+        },
+        ```
+
+    - **injectLoadHook**
+      - Type: `Boolean`
+        - Default: `false`
+        - By default Sentry will be lazy loaded once `window.onNuxtReady` is called. If you want to explicitly control when Sentry will be loaded you can set `injectLoadHook: true`. The module will inject a `$sentryLoad` method into the Nuxt.js context which you need to call once you are ready to load Sentry
+        ```js
+        // layouts/default.vue
+        ...
+        mounted() {
+          // Only load Sentry after initial page has fully loaded
+          // (this example should behave similar to using window.onNuxtReady though)
+          this.$nextTick(() => this.$sentryLoad())
+        }
+        ```
+
+    - **mockApiMethods**
+      - Type: `Boolean` or `Array`
+        - Default `true`
+        - Which API methods from `@sentry/browser` should be mocked. You can use this to only mock methods you really use.
+        - This option is ignored when `injectMock: false`
+        - If `mockApiMethods: true` then all available API methods will be mocked
+        > If `injectMock: true` then _captureException_ will always be mocked for use with the window.onerror listener
+        ```js
+        // nuxt.config.js
+        sentry: {
+          lazy: {
+            mockApiMethods: ['captureMessage']
+          }
+        }
+
+        // pages/index.vue
+        mounted() {
```

Comment:
Could use the `[filepath]` syntax for file name and maybe split into two code blocks.